File: artifacts/littlefs/decompose/src/lfs_file_config.c
struct lfs_file_config
{
  void *buffer;
  struct lfs_attr *attrs;
  lfs_size_t attr_count;
}
/*** DEPENDENCIES:
struct lfs_attr
{
  uint8_t type;
  void *buffer;
  lfs_size_t size;
}
----------------------------
typedef uint32_t lfs_size_t
----------------------------
***/

File: artifacts/littlefs/translate/src/lfs_file_config.rs
use crate::*;
use std::fs::File;

#[derive(Clone)]
pub struct LfsFileConfig {
    pub buffer: Option<Vec<u8>>,
    pub attrs: Option<Vec<LfsAttr>>,
    pub attr_count: crate::types::lfs_size_t,
}

